src/vs/workbench/api/browser/positron/mainThreadLanguageRuntime.ts

```diff
@@ -1018,7 +1095,7 @@ class ExtHostRuntimeClientInstance<Input, Output>
 	 *
 	 * @param request The request to send to the server.
 	 * @param timeout Timeout in milliseconds after which to error if the server
-	 *   does not respond.
+	 *   does not respond. If <= 0 the promise will never timeout.
```

Comment:
I think it would be clearer to use `undefined` to signal no timeout rather than passing a special/arbitrary numeric value.

Reply:
Done in https://github.com/posit-dev/positron/pull/7286/commits/8b30cb7d58270e848da22a5b8deafadad30429b0